Toolkit/ArcGISToolkit/AR/ArcGISARView.swift

```diff
@@ -180,17 +197,24 @@ public class ArcGISARView: UIView {
     /// - Parameter initialTransformation: The initial transformation for originCamera offset.
     /// - Returns: Whether setting the `initialTransformation` succeeded or failed.
     public func setInitialTransformation(initialTransformation: AGSTransformationMatrix) -> Bool {
-        fatalError("setInitialTransformation(initialTransformation:) has not been implemented")
+        _initialTransformation = initialTransformation
+        return true
     }
     
     /// Sets the initial transformation used to offset the originCamera.  The initial transformation is based on an AR point determined via existing plan hit detection from `screenPoint`.  If an AR point cannot be determined, this method will return `false`.
     ///
     /// - Parameter screenPoint: The screen point to determine the `initialTransformation` from.
     /// - Returns: Whether setting the `initialTransformation` succeeded or failed.
-    public func setInitialTransformation(screenPoint: CGPoint) -> Bool {
-        fatalError("setInitialTransformationPlacingOriginOnPlane(screenPoint:) has not been implemented")
-    }
+    public func setInitialTransformation(_ screenPoint: CGPoint) -> Bool {
```

Comment:
This looks a bit confusing at the call site:

```swift
view.setInitialTransformation(CGPoint(...))
```

The method name makes it sound like we're setting a transformation, but the parameter type is a point. Perhaps a parameter label is in order?

```suggestion
    public func setInitialTransformation(using screenPoint: CGPoint) -> Bool {
```